provider/vsphere/internal/vsphereclient/createvm.go

```diff
@@ -398,7 +398,7 @@ func (c *Client) extendVMRootDisk(
 	sizeMB uint64,
 	taskWaiter *taskWaiter,
 ) error {
-	disk, backing, err := c.getDiskWithFileBacking(ctx, vm)
+	disk, _, err := c.getDiskWithFileBacking(ctx, vm)
```

Comment:
Might as well change this method to `getDisk` and drop the backing return value, it doesn't look like anything else uses it.